### PR TITLE
sculpt: whitelist iwlwifi-9260 firmware and enable routing

### PR DESCRIPTION
**Pull request by a-dmg**
Dear Genodians,

I am following up with our discussion on the ML about the iwlwifi-9260-firmware.

Here is a commit to enable the wifi driver with the 'iwlwifi-9260-th-b0-jf-b0-34.ucode' firmware to be used with Sculpt.

Cheers,
Alice

### CODE REVIEW

**Comment by chelmuth** [comment]
Depends on cnuke/dde_linux_firmware#1.

File: repos/dde_linux/ports/linux-firmware.port [hunk 1]
```diff
@@ -2,7 +2,7 @@ LICENSE   := mixed
 VERSION   := 1
 DOWNLOADS := fw.archive
 
-FW_REV  := b7e07499ca264b5535c5c2b066d8007d92dfe43e
+FW_REV  := b8472dbf35b3a67ad9a00a05182e8ff39400c5c1
```
Comment (chelmuth) [inline repos/dde_linux/ports/linux-firmware.port:5]: This commit ID is not in https://github.com/cnuke/dde_linux_firmware but the contents should be equivalent to b7e07499ca264b5535c5c2b066d8007d92dfe43e.

@cnuke would you mind to fast-forward your master to make clear what's the canonical head commit?

File: repos/gems/src/app/sculpt_manager/runtime/wifi_drv.cc [hunk 2]
```diff
@@ -86,6 +86,8 @@ void Sculpt::gen_wifi_drv_start_content(Xml_generator &xml)
 		gen_parent_rom_route(xml, "iwlwifi-8265-36.ucode");
 		gen_parent_rom_route(xml, "iwlwifi-9000-pu-b0-jf-b0-34.ucode");
 		gen_parent_rom_route(xml, "iwlwifi-9000-pu-b0-jf-b0-46.ucode");
+		gen_parent_rom_route(xml, "iwlwifi-9260-th-b0-jf-b0-34.ucode");
+		gen_parent_rom_route(xml, "iwlwifi-9260-th-b0-jf-b0-46.ucode");
```
Comment (chelmuth) [inline repos/gems/src/app/sculpt_manager/runtime/wifi_drv.cc:90]: This contradicts the change below (please see my next comment).

File: repos/pc/src/lib/wifi/firmware.cc [hunk 3]
```diff
@@ -45,6 +45,8 @@ Firmware_list fw_list[] = {
 	{ "iwlwifi-9000-pu-b0-jf-b0-34.ucode", 2678284, nullptr },
 	{ "iwlwifi-9000-pu-b0-jf-b0-36.ucode", 2678284, "iwlwifi-9000-pu-b0-jf-b0-34.ucode" },
 	{ "iwlwifi-9000-pu-b0-jf-b0-46.ucode", 1514876, nullptr },
+	{ "iwlwifi-9260-th-b0-jf-b0-34.ucode", 2678092, nullptr },
+	{ "iwlwifi-9260-th-b0-jf-b0-46.ucode", 2678092, "iwlwifi-9260-th-b0-jf-b0-34.ucode" },
```
Comment (chelmuth) [inline repos/pc/src/lib/wifi/firmware.cc:49]: This contradicts the change above. You define *iwlwifi-9260-th-b0-jf-b0-46.ucode* is equivalent to *iwlwifi-9260-th-b0-jf-b0-34.ucode*, which will then be requested as ROM. Also, [cnuke/](https://github.com/cnuke/dde_linux_firmware.git) does not contain *iwlwifi-9260-th-b0-jf-b0-46.ucode* on the topic branch.

File: repos/dde_linux/ports/linux-firmware.port (hunk 1 above)
Comment (cnuke) [reply repos/dde_linux/ports/linux-firmware.port:5]: @chelmuth done

**Comment by cnuke** [comment]
I am going to close the pull request as it is solved by #4807.

**Pull request closed without merging**